data_provenance/to-dolma.py

```diff
@@ -151,6 +178,12 @@ def main(args):
     )
     to_dolma(examples, args.outdir, args.filename, args.shard_size)
 
+    valid_ids = set(include_df["Dataset ID"])
+    unique_doc_ids = get_unique_ids(args.outdir)
```

Comment:
This function has a confusing name, dolma ids are example level, but you're actually using dataset level ids to make sure each dataset is processed? We should make that clearer

Reply:
Adjusted names to be more clear that they are DPI related